ide/bcpg/nbproject/project.properties

```diff
@@ -14,5 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-release.external/bcpg-jdk15on-1.64.jar=modules/bcpg.jar
+javac.compilerargs=-Xlint -Xlint:-serial
+javac.source=1.8
```

Comment:
Are there sources in the module?

Reply:
You are right, there are no sources. Thanks.